include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -358,6 +358,11 @@ ORT_API_STATUS(OrtGetTensorMemSizeInBytesFromTensorProto, _In_ const void* input
  */
 ORT_API(const OrtTensorTypeAndShapeInfo*, OrtCastTypeInfoToTensorInfo, _In_ OrtTypeInfo*);
 
+/**
+ * Return OnnxType from OrtTypeInfo
+ */
+ORT_API(enum ONNXType, OrtOnnxTypeFromTypeInfo, _In_ OrtTypeInfo*);
```

Comment:
can OrtTypeInfo* be a const ptr? Why not?

Reply:
yes, it can/should. Done.